docs/Simulators.md

```diff
@@ -4,14 +4,14 @@ Inductiva API has available several open-source simulators ready to use. Users w
 No installation or management of the simulators is required, no need to worry about hardware or software dependencies. Inductiva API takes care of all of that for you.
 
 The simulators currently available are all open-source and have their dedicated documentation:
-- [SPlisHSPlasH](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH)
-- [DualSPHysics](https://dual.sphysics.org/)
-- [OpenFOAM](https://www.openfoam.com/)
-- [SWASH](https://swash.sourceforge.io/)
-- [XBeach](https://oss.deltares.nl/web/xbeach/)
-- [Reef3D](https://reef3d.wordpress.com/)
-- [GROMACS](https://www.gromacs.org/)
-- [FDS](https://pages.nist.gov/fds-smv/)
+- [SPlisHSPlasH](https://github.com/inductiva/inductiva/blob/main/docs/simulators/SPlisHSPlasH.md)
```

Comment:
This then, needs to point to the tab for each simulator in the readthedocs.

Reply:
Seems like the integration with the pull requests here in github and readthedocs is working ok, you can see the result  of this PR at:
https://inductiva-research-labs-inductiva--966.com.readthedocs.build/en/966/Simulators.html

and indeed the links will point you to outside readthedocs (they will bring you to github)....

Reply:
Basically, I think we should assume that in the github repo people will only look the main README file, but they will not be there navigating to other MD files with documentation. That kind of browsing or navigation is much better in readthedocs, with the sidebar, etc.

For example: pytorch has a huge README file, with lots of links, but it either links to subsections on the README file (just jumps to a section below) or points to their main documentation website:
https://github.com/pytorch/pytorch

wdyt @IvanPombo @AugustoPeres and @sergiomsantos ?

Reply:
I like the idea of having a single md file with several sections. The links above (excluding splishsplash) are fine as they point to the official pages of the simulators

Reply:
I think the links should be pointing to the tabs inside the READTHEDOCS with the specifications of how each simulator works in the API. There, we have a link pointing to the main documentation/webpage of the simulator. I think the purpose is for them to be immersed on the API and not to take them out to other pages. Most users, will already know the documentation page anyway :D

Reply:
Ok. I was just alerting to the fact that readthedocs already uses this MD file to generate one section, and these links will forward the user to github, instead of leaving the users in readthedocs. But you can merge and fix that in a separate PR if you prefer.

Reply:
I had a review that didn't push before that went exactly in this direction. These links should be relative and not absolute. Readthedocs should then mark them as relative also (I think).
Also, thy are all pointing to the main branch. They should all point to the relevant md files in whatever branch they are.